draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -164,55 +164,58 @@ The YANG data model defined in this document conforms to the Network Management
 
 ## Terminology
 
-  The following terms are defined in {{!RFC7950}} and are not
-  redefined here:
+The following terms are defined in {{!RFC7950}} and are not redefined here:
+- client
+- server
+- augment
+- data model
+- data node
 
-  *  client
+The following terms are defined in {{!RFC6241}} and are not redefined here:
+- configuration data
+- state data
 
-  *  server
+The following terms are defined in {{IANA_YANG}} and are not redefined here:
+- backplane
+- battery
+- container
+- cpu
+- chassis
+- fan
+- module
+- port
+- power supply
+- sensor
+- stack
+- storage device
 
-  *  augment
+> Note that the definition of port component in {{IANA_YANG}} needs to be refined in future version of this document.
 
-  *  data model
+> TBD: Recap the concept of chassis/slot/component/board/... in {{TMF_SD2-20}}.
```

Comment:
I think it would be better to create a related issue about that instead to have this note in the draft. Just a suggestion.

Reply:
This should be resolved by the appendix which is work-in-progress in PR #66